### PR TITLE
[etckeeper] Install etckeeper from backports

### LINKED ISSUE #1560: [etckeeper] Use py3-compatible version from buster-backports
`debops.etckeeper` installs etckeeper from the Debian Stable repository, which is currently at version 1.18.10-1 for Debian 10. This package however depends on Python 2. The role does not install etckeeper by default when a Python 2 environment is not present on the host (see [etckeeper__enabled](https://docs.debops.org/en/master/ansible/roles/etckeeper/defaults/main.html#envvar-etckeeper__enabled)).

I propose that, on Debian 10, we install etckeeper from buster-backports. The package in that repository (version 1.18.14-1~bpo10+1) depends on Python 3 instead. This should allow default installation of etckeeper on all hosts, not just the hosts that have a Python 3 environment.

Let me know what you think of this and I'll put some effort in.

### PR DESCRIPTION
**Pull request by imrejonk**
Closes #1560

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -194,6 +194,13 @@ LDAP
 
 - The DHCP Relay Agent functionality has been moved to :ref:`debops.dhcrelay`.
 
+:ref:`debops.etckeeper` role
+''''''''''''''''''''''''''''
+
+- The role now installs etckeeper on all hosts by default, not just on hosts
+  that have a Python 2 environment. Etckeeper is also installed from
```
Comment (ypid) [inline CHANGELOG.rst]:
```suggestion
  that have a Python 2 environment. etckeeper is also installed from
```

**Review by ypid — CHANGES_REQUESTED** [review]
LGTM. I only ask for some style changes:

* Consistent form of "etckeeper". It seems the authors want it to be always written as lowercase. Ref: https://etckeeper.branchable.com/README/
* Consistent commit message. Use `[etckeeper]` instead of `[etc_keeper]`.

**Comment by ypid** [comment]
And not to forget `docs/ansible/roles/etckeeper/getting-started.rst`.

File: CHANGELOG.rst (hunk 1 above)
Comment (imrejonk) [reply CHANGELOG.rst]: Crap, I keep writing etc_keeper... Thanks for spotting this.

**Comment by imrejonk** [comment]
Docs are updated as well now

**Review by ypid — APPROVED** [review]
LGTM. Thank you. So the Python 3 only future is near :)

**Review by drybjed — APPROVED** [review]
Sounds good. I'll fix the Changelog conflict and add the `etckeeper` `apt_preferences` entry in the common playbook.

**Pull request merged**